### ISSUE TITLE
DevTools failed to load source map: Could not load content for http://localhost:3000/@vite/client.mjs.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

### ISSUE THREAD

**Issue by kazupon** [intlify/bundle-tools#82]
We are missing a source map with `@intlify/vite-plugin-vue-i18n@3.2.1` on dev mode

### Console Error
DevTools failed to load source map: Could not load content for http://localhost:3000/@vite/client.mjs.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

### Steps to reproduce
1. Create a new project with `npm init vite@latest my-vue-app -- --template vue`
2. Install vue-i18n with `npm i @intlify/vite-plugin-vue-i18n vue-i18n`
3. Add i18n plugin to `vite.config.js`
4. Run dev mode `npm run dev` to inspect

**package.json**
```.json
{
  "name": "my-vue-app",
  "version": "0.0.0",
  "scripts": {
    "dev": "vite",
    "build": "vite build",
    "preview": "vite preview"
  },
  "dependencies": {
    "@intlify/vite-plugin-vue-i18n": "^3.2.1",
    "vue": "^3.2.25",
    "vue-i18n": "^9.1.9"
  },
  "devDependencies": {
    "@vitejs/plugin-vue": "^2.0.1",
    "vite": "^2.7.12"
  }
}
```

**vite.config.js**
```.js
import { defineConfig } from 'vite'
import vue from '@vitejs/plugin-vue'
import i18n from '@intlify/vite-plugin-vue-i18n'

// https://vitejs.dev/config/
export default defineConfig({
  plugins: [vue(), i18n()],
})

```

### Environment
macOS 12.1
Node 14.18.2
npm 8.3.1
Chrome 97
Safari 15.2

_I can see the inlined source map for http://localhost:3000/@vite/client when I don't have installed the plugin, that's why I suspect that it's related._

I don't know how I can help to fix it, I hope somebody can address me.

_Originally posted by @fcomrqz in https://github.com/intlify/bundle-tools/issues/1#issuecomment-1013841297_

**Comment by Ragura** [comment]
Not with the newest version. You need to downgrade to version 3.1.1 to prevent it.
I tried looking into fixing it, but couldn't figure it out. I understand that @kazupon doesn't immediately see a way to resolve it either, so it would be nice if the faulty source map implementation could just be reverted, then we can upgrade normally again.

**Comment by kazupon** [comment]
resolved with v3.3.1
